tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "module": "commonjs",
     "noImplicitAny": true,
     "outDir": "build",
-    "rootDir": "src",
+    "rootDirs": ["src"],
```

Comment:
Why this change?  `rootDir` and `rootDirs` are different things, and I'm not sure how it affects the build output...

Reply:
Fixed to previous state